modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java

```diff
@@ -390,7 +409,8 @@ protected void exportImportPortlet(
 						TYPE_PUBLISH_PORTLET_LOCAL,
 					settingsMap);
 
-		ExportImportThreadLocal.setPortletStagingInProcess(true);
+		ExportImportThreadLocal.setPortletStagingInProcess(staging);
+		ExportImportThreadLocal.setPortletExportInProcess(exporting);
```

Comment:
@joseabelenda can you order it alphabetically?

```
ExportImportThreadLocal.setPortletExportInProcess(..);
ExportImportThreadLocal.setPortletStagingInProcess(...);
```

Reply:
Variable `exporting` was removed.